content/billing/managing-billing-for-your-github-account/connecting-an-azure-subscription.md

```diff
@@ -22,6 +22,8 @@ shortTitle: Connect an Azure subscription
 
 **Note:** If your enterprise account is on a Microsoft Enterprise Agreement, connecting an Azure subscription is the only way to use {% data variables.product.prodname_actions %} and {% data variables.product.prodname_registry %} beyond the included amounts, or to use {% data variables.product.prodname_github_codespaces %} and {% data variables.product.prodname_copilot_business_short %} at all.
 
+**Note:** If your organisation is a part of an enteprirse account, you will only be able to connect your Azure subscription to enteprise account and not to orgnisation account.
```

Comment:
Thanks @andrekolodochka! I think should actually be regular text outside the note, just above the `{% data reusables.enterprise.ghec-trial-azure %}` reusable. Could you move it up there? Thanks!

Reply:
enterprise account is only applicable to orgs on Enterprise plan. I intentionally included my changes in `{% ifversion ghec %}` block, I don't think placing it above `{% data reusables.enterprise.ghec-trial-azure %}` is the right option. Let me know what you think.

Reply:
We'll just need to create an additional `{% ifversion ghec %}` around the new line, which is totally fine! 😊